### PR TITLE
[IMP] mail plugins: gmail plugin updates

### PR DESCRIPTION
**Pull request by meng-odoo**
Updated to reflect how to download Gmail Plugin from Google Workspace Marketplace for Odoo Online users; added new images

Closes Odoo Task 3001713

### CODE REVIEW

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 1]
```diff
@@ -2,94 +2,143 @@
...
+Next, in the ZIP file, go to :menuselection:`mail-client-extensions-master --> gmail`, and open the
+file called :guilabel:`README`. Follow the instructions in the :guilabel:`README` file to push the
+Gmail Plugin files as a Google Project.
+
+.. note::
+   Your computer must be able to run Linux commands in order to follow the instructions on the :guilabel:`README` file.
+
+Share the Google Project with the Gmail account that you wish to connect with Odoo. Then click
+:guilabel:`Publish` and :guilabel:`Deploy from manifest`. Lastly, click
+:guilabel:`Install the add-on` to install the Gmail Plugin.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:113]:
```suggestion
After that, share the Google Project with the Gmail account that the user wishes to connect with
Odoo. Then, click :guilabel:`Publish` and :guilabel:`Deploy from manifest`. Lastly, click
:guilabel:`Install the add-on` to install the Gmail Plugin.
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 2]
```diff
@@ -2,94 +2,143 @@
...
+
+Configure your Odoo database
+----------------------------
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:116]:
```suggestion
Configure the Odoo database
---------------------------
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 3]
```diff
@@ -2,94 +2,143 @@
...
 
-#. The right-side panel can now display **Company Insights**. At the bottom, click on *Login*.
+The :guilabel:`Mail Plugin` feature must be enabled in your Odoo database in order to use the Gmail
+Plugin. Go to :menuselection:`Settings --> General Settings`. Under :guilabel:`Integrations`,
+activate :guilabel:`Mail Plugin`, and then click :guilabel:`Save`.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:120]:
```suggestion
The :guilabel:`Mail Plugin` feature must be enabled in the Odoo database in order to use the Gmail
Plugin. To enable the feature, go to :menuselection:`Settings --> General Settings`. Under
:guilabel:`Integrations`, activate :guilabel:`Mail Plugin`, and then click :guilabel:`Save`.
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 4]
```diff
@@ -2,94 +2,143 @@
...
 
-   .. image:: gmail/panel-login.png
-      :align: center
-      :alt: Logging in your Odoo database
+.. image:: gmail/mail-plugin-setting.png
+   :align: center
+   :alt: Mail Plugin feature in Settings
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:124]:
```suggestion
   :alt: The Mail Plugin feature in the Settings.
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 5]
```diff
@@ -2,94 +2,143 @@
...
 
-   .. note::
-      Only a limited amount of *Company Insights* (*Lead Enrichment*) requests are available as a
-      trial. This feature requires :ref:`prepaid credits <mail_plugins/pricing>`.
+Configure your Gmail inbox
+--------------------------
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:127]:
```suggestion
Configure the Gmail inbox
-------------------------
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 6]
```diff
@@ -2,94 +2,143 @@
...
 
-#. Enter your Odoo database URL and click on *Login*.
+In your Gmail inbox, a purple Odoo icon will now appear on the right sidebar. Click on the Odoo
+icon to open up the Odoo plugin window. Then, click on any email in your inbox. Click
+:guilabel:`Authorize Access` in the plugin window to grant Odoo access to your Gmail inbox.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:131]:
```suggestion
In the Gmail inbox, a purple Odoo icon is now visible on the right sidebar. Click on the Odoo icon
to open up the Odoo plugin window. Then, click on any email in the inbox. Next, click
:guilabel:`Authorize Access` in the plugin window to grant Odoo access to the Gmail inbox.
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 7]
```diff
@@ -2,94 +2,143 @@
...
 
-   .. image:: gmail/database-url.png
-      :align: center
-      :alt: Entering your Odoo database URL
+.. image:: gmail/authorize-access.png
+   :align: center
+   :alt: Authorize Access button in the right sidebar of the Odoo plugin panel
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:135]:
```suggestion
   :alt: The Authorize Access button in the right sidebar of the Odoo plugin panel.
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 8]
```diff
@@ -2,94 +2,143 @@
...
 
-#. If you aren't logged into your database, enter your credentials.
+Next, click :guilabel:`Login`. Enter the URL of the Odoo database that you wish to connect with
+your Gmail inbox, and log in to the database.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:138]:
```suggestion
Next, click :guilabel:`Login`. Enter the URL of the Odoo database that the user wishes to connect to
the Gmail inbox, and log in to the database.
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 9]
```diff
@@ -2,94 +2,143 @@
...
 
-#. Click on *Allow* to let the Gmail Plugin connect to your database.
+.. note::
+   Use the general URL for the database, not the URL of a specific page in the database. For example, use `https://stealthywood.odoo.com`, not `https://stealthywood.odoo.com/web#cids=1&action=menu`.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:141]:
```suggestion
   Use the general URL for the database, not the URL of a specific page in the database. For example,
   use `https://mycompany.odoo.com`, not `https://mycompany.odoo.com/web#cids=1&action=menu`.
```

Also, make sure the first letter of the second line of the note lines up with the first letter of the first line!

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 10]
```diff
@@ -2,94 +2,143 @@
...
 
-   .. image:: gmail/odoo-permission.png
-      :align: center
-      :alt: Allowing the Gmail Plugin to connect to a database
\ No newline at end of file
+Then, click :guilabel:`Allow` to let Gmail access your Odoo database. The browser will then show a
+“Success!” message. Close the window. Your Gmail inbox and Odoo database are now connected.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:144]:
```suggestion
Finally, click :guilabel:`Allow` to let Gmail access the Odoo database. The browser will then show a
:guilabel:`Success!` message. Close the window. The Gmail inbox and Odoo database are now connected.
```

**Review by mivu-odoo — CHANGES_REQUESTED** [review]
Hello @meng-odoo!

Great job on your first PR! I went through it and found some edits that need to be made to remove the use of second-person pronouns (you, yours), as well sentences that could use transition words for better flow.

I also noticed two image files were quite large `gh-download-zip.png` and `google-workspace-marketplace.png`. I think they can be ran through pngquant (if you haven't already) to compress them further. I can show you how to remove the old image files and replace them with the new, compressed versions if you would like, just let me know!

When you finish the revisions, please tag me again for another look. Thank you :smile_cat:

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 11]
```diff
@@ -2,94 +2,143 @@
 Gmail Plugin
 ============
 
-Configuration
-=============
+The Gmail Plugin integrates your Odoo database with your Gmail inbox.
```
[the remark this answers is not in the document]
Comment (meng-odoo) [reply content/applications/productivity/mail_plugins/gmail.rst:5]: Let's change this sentence to the below suggestion.
```suggestion
The *Gmail Plugin* integrates an Odoo database with a Gmail inbox, so users can keep track of all their work between Gmail and Odoo, without losing any information.
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 12]
```diff
@@ -2,94 +2,143 @@
...
 
-First, you need to enable the *Mail Plugin* feature in your database. Go to :menuselection:`Settings
---> General Settings --> Integrations`, enable *Mail Plugin*, and *Save* the configuration.
+First, make sure you are logged in to the Gmail account that you wish to connect with Odoo.
 
-.. _mail-plugin/gmail/install-plugin:
+From your Gmail inbox, click the plus sign (“+”) on the right sidebar to get add-ons. Use the
+search bar to search for “Odoo” and locate the :guilabel:`Odoo Inbox Addin`. Or, go directly to
+the :guilabel:`Odoo Inbox Addin` page on `Google Workspace Marketplace
+<https://workspace.google.com/marketplace/app/odoo_inbox_addin/873497133275>`_.
```
[the remark this answers is not in the document]
Comment (meng-odoo) [reply content/applications/productivity/mail_plugins/gmail.rst:21]: You just need to click the little arrow at the bottom right of your screen to show your side panel. I could include a picture of this in the doc to help people find it.

[the remark this answers is not in the document]
Comment (mivu-odoo) [reply content/applications/productivity/mail_plugins/gmail.rst:21]: Yes, include an image and also mention that if it isn't visible, just click the arrow icon in the bottom right corner.

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 13]
```diff
@@ -2,94 +2,160 @@
...
-------------------
+Install the Gmail Plugin
+------------------------
 
-First, you need to enable the *Mail Plugin* feature in your database. Go to :menuselection:`Settings
---> General Settings --> Integrations`, enable *Mail Plugin*, and *Save* the configuration.
+First, log in to the Gmail account that the user wishes to connect to Odoo.
 
-.. _mail-plugin/gmail/install-plugin:
+From the Gmail inbox, click the plus sign icon on the right side panel to get add-ons. If the side panel is not visible, click on the arrow icon at the bottom right corner of the inbox to reveal it.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:19]:
```suggestion
From the Gmail inbox, click the plus sign icon on the right side panel to get add-ons. If the side
panel is not visible, click on the arrow icon at the bottom right corner of the inbox to reveal it.
```

Remember to break the line before the 100th character!

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 14]
```diff
@@ -2,94 +2,160 @@
...
+   :align: center
+   :alt: The Authorize Access button in the right sidebar of the Odoo plugin panel.
+
+Next, click :guilabel:`Login`. Then, enter the URL of the Odoo database that the user wishes to
+connect to the Gmail inbox, and log in to the database.
+
+.. note::
+   Use the general URL for the database, not the URL of a specific page in the database. For
+   example, use `https://mycompany.odoo.com`, not
+   `https://mycompany.odoo.com/web#cids=1&action=menu`.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:69]:
```suggestion
   example, use `https://mycompany.odoo.com`, not `https://mycompany.odoo.com/web#cids=1&action=menu`.
```

This line break was a little early! You can fit all of this into one line and it should be under 100 characters.

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 15]
```diff
@@ -2,94 +2,160 @@
...
+
+Install the Gmail Plugin
+------------------------
 
-      .. image:: gmail/legacy-editor.png
-         :align: center
-         :alt: Using the legacy editor in the Gmail Plugin Apps Script project
+First, access the `GitHub repository <https://github.com/odoo/mail-client-extensions>`_ for the
+Odoo Mail Plugins. Next, click on the green :guilabel:`Code` button. Then click
+:guilabel:`Download ZIP` to download the Mail Plugin files onto the computer.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:96]:
```suggestion
Odoo Mail Plugins. Next, click on the green :guilabel:`Code` button. Then, click :guilabel:`Download
ZIP` to download the Mail Plugin files onto the computer.
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 16]
```diff
@@ -2,94 +2,160 @@
...
+   :align: center
+   :alt: Download the ZIP file from the Odoo GitHub repository for Mail Plugins.
 
-   .. image:: gmail/install-add-on.png
-      :align: center
-      :alt: Installing the Gmail Plugin from the Apps Script project
+Open the ZIP file on the computer. Then, go to
+:menuselection:`mail-client-extensions-master --> gmail --> src --> views`, and open the
+:guilabel:`login` TS file using any text editor software, such as Notepad (Windows), TextEdit
+(Mac), or Visual Studio Code.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:105]:
```suggestion
Open the ZIP file on the computer. Then, go to :menuselection:`mail-client-extensions-master
--> gmail --> src --> views`, and open the :guilabel:`login` TS file using any text editor software,
such as Notepad (Windows), TextEdit (Mac), or Visual Studio Code.
```

**Review by mivu-odoo — CHANGES_REQUESTED** [review]
Hi @meng-odoo!

One more round of revisions, we're almost there!

These suggestions have to do with line breaks. Some lines were broken a little too late (over 100 characters) or a little too early (under 100 characters, where more words could've fit on the line). When you're ready, tag me in for a final look. Thank you :smile_cat:

File: content/applications/productivity/mail_plugins/gmail.rst (hunk 14 above)
Comment (meng-odoo) [reply content/applications/productivity/mail_plugins/gmail.rst:69]: It's more than 100 characters, the URL gets cut off.

File: content/applications/productivity/mail_plugins/gmail.rst (hunk 15 above)
Comment (meng-odoo) [reply content/applications/productivity/mail_plugins/gmail.rst:96]: It's over 100 characters if I cut the line off after "Download", and the `guilabel` doesn't work if I cut it off after `:guilabel:`, so I cut it off after "click".

File: content/applications/productivity/mail_plugins/gmail.rst (hunk 16 above)
Comment (meng-odoo) [reply content/applications/productivity/mail_plugins/gmail.rst:105]: This works, thanks.

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 17]
```diff
@@ -2,94 +2,160 @@
...
+Then, use the search bar to search for `Odoo` and locate the :guilabel:`Odoo Inbox Addin`.
+
+.. image:: gmail/google-workspace-marketplace.png
+   :align: center
+   :alt: Odoo Inbox Addin on Google Workspace Marketplace.
+
+Or, go directly to the :guilabel:`Odoo Inbox Addin` page on the `Google Workspace Marketplace
+<https://workspace.google.com/marketplace/app/odoo_inbox_addin/873497133275>`_.
+
+Once the plugin is located, click :guilabel:`Install`. Then click :guilabel:`Continue` to start
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:35]:
```suggestion
Once the plugin is located, click :guilabel:`Install`. Then, click :guilabel:`Continue` to start
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 18]
```diff
@@ -2,94 +2,160 @@
...
+installation was successful.
+
+Configure the Odoo database
+---------------------------
 
-#. Open the `Gmail Plugin Apps Script project
-   <https://script.google.com/d/1n7cxtaR4fGXKcP0RwinNQmL8S4FhVqpo-ZZ_cUAhYuuDpZAP_CnHE_7q/edit>`_.
+The :guilabel:`Mail Plugin` feature must be enabled in the Odoo database in order to use the Gmail
+Plugin. To enable the feature, go to :menuselection:`Settings --> General Settings`. Under
+:guilabel:`Integrations`, activate :guilabel:`Mail Plugin`, and then click :guilabel:`Save`.
```
Comment (mivu-odoo) [inline content/applications/productivity/mail_plugins/gmail.rst:47]:
```suggestion
Plugin. To enable the feature, go to :menuselection:`Settings --> General Settings`. Under the
:guilabel:`Integrations` section, activate :guilabel:`Mail Plugin`, and then click :guilabel:`Save`.
```

**Review by mivu-odoo — APPROVED** [review]
Hello @meng-odoo!

Thank you for the revisions! You did a great job on this PR!

I found two small edits, so feel free to push those and then go straight to tagging ZST for final technical review, no need to tag me again. Thank you :smile_cat:

**Comment by meng-odoo** [comment]
Thank you @mivu-odoo!
Hi @StraubCreative, this doc is ready for your technical review. Thanks :)

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 19]
```diff
@@ -2,94 +2,162 @@
...
+Open the ZIP file on the computer. Then, go to :menuselection:`mail-client-extensions-master -->
+gmail --> src --> views`, and open the :guilabel:`login` TS file using any text editor software,
+such as Notepad (Windows), TextEdit (Mac), or Visual Studio Code.
 
-.. _mail-plugin/gmail/configure-mailbox:
+Delete the following three lines of text from the :guilabel:`login` TS file:
 
-Configure your Gmail mailbox
-----------------------------
+.. code-block:: python
```
Comment (StraubCreative) [inline content/applications/productivity/mail_plugins/gmail.rst:110]: Remove `python` and the CI error will resolve (this isn't actually python per se as it's in a `.ts` file not a `.py` file).
```suggestion
.. code-block::
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 20]
```diff
@@ -2,94 +2,162 @@
...
 
-   .. image:: gmail/install-add-on.png
-      :align: center
-      :alt: Installing the Gmail Plugin from the Apps Script project
+Open the ZIP file on the computer. Then, go to :menuselection:`mail-client-extensions-master -->
+gmail --> src --> views`, and open the :guilabel:`login` TS file using any text editor software,
+such as Notepad (Windows), TextEdit (Mac), or Visual Studio Code.
 
-.. _mail-plugin/gmail/configure-mailbox:
+Delete the following three lines of text from the :guilabel:`login` TS file:
```
Comment (StraubCreative) [inline content/applications/productivity/mail_plugins/gmail.rst:108]: Use `:file:` tag here:
```suggestion
Delete the following three lines of text from the :file:`login.ts` file:
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 21]
```diff
@@ -2,94 +2,162 @@
...
-   click on *Close*.
+.. image:: gmail/gh-download-zip.png
+   :align: center
+   :alt: Download the ZIP file from the Odoo GitHub repository for Mail Plugins.
 
-   .. image:: gmail/install-add-on.png
-      :align: center
-      :alt: Installing the Gmail Plugin from the Apps Script project
+Open the ZIP file on the computer. Then, go to :menuselection:`mail-client-extensions-master -->
+gmail --> src --> views`, and open the :guilabel:`login` TS file using any text editor software,
```
Comment (StraubCreative) [inline content/applications/productivity/mail_plugins/gmail.rst:105]: Use `:file:` tag here:
```suggestion
gmail --> src --> views`, and open the :file:`login.ts` file using any text editor software,
```

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 22]
```diff
@@ -2,94 +2,143 @@
...
 
-   .. image:: gmail/choose-account.png
-      :align: center
-      :alt: Choosing your Google account
+First, access the `GitHub repository <https://github.com/odoo/mail-client-extensions>`_ for the
+Odoo Mail Plugins. Click on the green :guilabel:`Code` button, then click :guilabel:`Download ZIP`
+to download the Mail Plugin files onto your computer.
 
-#. Allow the Gmail Plugin to access some of your data.
+.. image:: gmail/gh-download-zip.png
```
[the remark this answers is not in the document]
Comment (StraubCreative) [reply content/applications/productivity/mail_plugins/gmail.rst:100]: Unresolving. This file is still too big.

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 23]
```diff
@@ -2,94 +2,143 @@
...
 
-.. _mail-plugin/gmail/install-plugin:
+From your Gmail inbox, click the plus sign (“+”) on the right sidebar to get add-ons. Use the
+search bar to search for “Odoo” and locate the :guilabel:`Odoo Inbox Addin`. Or, go directly to
+the :guilabel:`Odoo Inbox Addin` page on `Google Workspace Marketplace
+<https://workspace.google.com/marketplace/app/odoo_inbox_addin/873497133275>`_.
 
-Install the Gmail Plugin
-------------------------
+.. image:: gmail/google-workspace-marketplace.png
```
[the remark this answers is not in the document]
Comment (StraubCreative) [reply content/applications/productivity/mail_plugins/gmail.rst:28]: Unresolving. This image is still too big.

File: content/applications/productivity/mail_plugins/gmail.rst [hunk 24]
```diff
@@ -2,94 +2,162 @@
...
---> General Settings --> Integrations`, enable *Mail Plugin*, and *Save* the configuration.
+First, log in to the Gmail account that the user wishes to connect to Odoo.
 
-.. _mail-plugin/gmail/install-plugin:
+From the Gmail inbox, click the plus sign icon on the right side panel to get add-ons. If the side
+panel is not visible, click on the arrow icon at the bottom right corner of the inbox to reveal it.
 
-Install the Gmail Plugin
-------------------------
+.. image:: gmail/gmail-side-panel.png
```
Comment (StraubCreative) [inline content/applications/productivity/mail_plugins/gmail.rst:22]: This image is too big.

- Capture screenshot in HD (browser at ~1280x600px with ~120% zoom)
- Resize file to tablet/low-res desktop breakpoint b/w 768 and ~1024px (lowest you can go without having to squint at the content).
- Compress file using `pngquant` 
- Replace file locally using `git rm` and `git add`

**Review by StraubCreative — CHANGES_REQUESTED** [review]
Hi @meng-odoo 🙂 
Change requests below, with instructions.
Thanks! 

cc: @mivu-odoo

**Comment by meng-odoo** [comment]
Hi @StraubCreative, I made all your requested changes. I'm still getting a failed code check that says "trailing whitespace" but I double checked every line and I don't see it. Please let me know if I should make any more changes. Thanks!

**Review by StraubCreative — APPROVED** [review]
Thanks for making all of the changes @meng-odoo, nice job 👍 
Everything looks good to me. 

I combed for whitespaces and did another rebase on [4625793](https://github.com/odoo/documentation/pull/2773/commits/4625793038b89dba1923736a4c650ec9854efa6c), but still couldn't the issue the CI check error, so I'm going to push to DR and maybe they can help.

**Comment by StraubCreative** [comment]
Hi @odoo/doc-review 👋 
This PR is ready for review.
Please advise we couldn't find the trailing whitespace the CI check claimed was there. 
We combed it twice and rebased twice as well. Can you help? TIA 🙏

**Comment by Feyensv** [comment]
> Hi @odoo/doc-review wave This PR is ready for review. Please advise we couldn't find the trailing whitespace the CI check claimed was there. We combed it twice and rebased twice as well. Can you help?

It means that there was some empty spaces left in your changes, though it's difficult to see unless you have a dedicated extension in your text editor:
image

I've removed those empty spaces and pushed an updated commit.

Note that the CI check was supposed to give the line where the linting error was found, which wasn't the case, we'll see with the runbot team to make sure it's correctly shown in the future ;).

**Pull request closed without merging**